olsrd/patches/102-olsrd-prevent-storm-patches.patch

```diff
@@ -0,0 +1,83 @@
+From 4dabd94d598dd893aaaffbd71c315923c8827a14 Mon Sep 17 00:00:00 2001
+From: Nick Hainke <vincent@systemli.org>
+Date: Wed, 22 Jun 2022 14:08:04 +0200
+Subject: [PATCH] olsrd: prevent storm patches
+
+As described in the PR:
+
+  Limit the positive sequence number difference which is considered valid,
+  and prevent network storms.
+  Source: https://github.com/aredn/aredn_packages/pull/5
```

Comment:
Next time, I'd prefer if you just add suffix .patch to the URL
( = https://github.com/aredn/aredn_packages/pull/5.patch ) and put the patch here. :)

Reply:
Yeah. I was confused that this is not a patch against olsr, but a patch against a fork. I wanted to replace the patch with a upstream patch and asked the developer to add a PR in the original olsr repository:
https://github.com/OLSR/olsrd/issues/106#issuecomment-1162773661

However, the person did not respond. :/ So I was not able to replace it for now.